src/lib/airtable/request.js

```diff
@@ -508,18 +509,22 @@ export const updateCustomer = async (id, recordUpdates) => {
 
 // NONGENERATED: Edit customer
 export const editCustomer = async (customer) => {
```

Comment:
I'd be careful with the variable naming here. This object is a `customer` with an added `customerUpdate` in it. Therefore, when you do `editCustomerInRedux(customer)`, the edited customer you added now looks different than all the other customers (the edited customer has a `customerUpdate` field that no other customer has).

You could:
1. pass them as 2 separate parameters initially and when you do the fetch you can `JSON.stringify({...customer, customerUpdate})` like how you did when calling `editCustomer`)
2. If you look at the custom backend endpoint, [here](https://github.com/calblueprint/meepanyar-node/pull/15/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R125), we really just separate out the data and do a `updateCustomer` and `createCustomerUpdate`. I think we could instead just separate out the data here and call `updateCustomer` and `createCustomerUpdate` here and not need the custom backend endpoint. I'm not sure if I'm missing something with this solution though, so let me know if you have concerns. 

If solution 2 doesn't have problems, I'd go with that since the frontend has built-in error handling (thanks to the offline library we're using), so we'll retry in the case either our network goes down or the backend is down etc. We could also just go with solution #1 for now and revisit solution #2 later. Your call